packages/venia-ui/lib/components/Region/region.js

```diff
@@ -49,7 +52,8 @@ export default Region;
 
 Region.defaultProps = {
     field: 'region',
-    label: 'State'
+    label: 'State',
+    optionValueKey: 'code'
```

Comment:
Add a comment here explaining that this is probably temporary while we wait on some backend bugs to be resolved. Add the MC ticket number too :)

Reply:
Added in 64cc0d5.